src/models/event-timeline-set.js

```diff
@@ -696,8 +696,9 @@ EventTimelineSet.prototype.compareEventOrdering = function(eventId1, eventId2) {
  * @param {String} eventType
  * The relation event's type, such as "m.reaction", etc.
  *
- * @returns {Relations}
- * A container for relation events.
+ * @returns {?Relations}
+ * A container for relation events, or undefined if either an event argument is invalid or
```

Comment:
the jsdoc bit looks right but the reasoning here seems wrong, in those two cases this method actually throws

Reply:
Yeah, that's true, just realized it. I'll correct the return comment asap